src/rfxgen.c

```diff
@@ -245,25 +195,6 @@ static void ShowCommandLineInfo(void);                      // Show command line
 static void ProcessCommandLine(int argc, char *argv[]);     // Process command line input
 #endif
 
-// Load/Save/Export data functions
-static WaveParams LoadWaveParams(const char *fileName);                 // Load wave parameters from file
-static void SaveWaveParams(WaveParams params, const char *fileName);    // Save wave parameters to file
-static void ResetWaveParams(WaveParams *params);                        // Reset wave parameters
-static Wave GenerateWave(WaveParams params);                            // Generate wave data from parameters
-
-// Sound generation functions
-static WaveParams GenPickupCoin(void);      // Generate sound: Pickup/Coin
-static WaveParams GenLaserShoot(void);      // Generate sound: Laser shoot
-static WaveParams GenExplosion(void);       // Generate sound: Explosion
-static WaveParams GenPowerup(void);         // Generate sound: Powerup
-static WaveParams GenHitHurt(void);         // Generate sound: Hit/Hurt
-static WaveParams GenJump(void);            // Generate sound: Jump
-static WaveParams GenBlipSelect(void);      // Generate sound: Blip/Select
-static WaveParams GenRandomize(void);       // Generate random sound
-static void WaveMutate(WaveParams *params); // Mutate current sound
-
-// Auxiliar functions
-static void DrawWave(Wave *wave, Rectangle bounds, Color color);    // Draw wave data using lines
```

Comment:
I think that function should stay in the main problem. `rfxgen.h` should be as much independent of raylib as possible.